spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -36,14 +36,6 @@
 
 	private final Publisher publisher = new Publisher();
 
-	public Subscriber getSubscriber() {
-		return this.subscriber;
-	}
-
-	public Publisher getPublisher() {
-		return this.publisher;
-	}
-
 	/** Overrides the GCP project ID specified in the Core module. */
```

Comment:
Please, consider to refactor JavaDocs in this class for the proper multi-line format.
This one looks ugly and breaks our code style.
Looks like @joaoandremartins didn't fix them all when we had and argument on the matter...